go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/alicebob/sqlittle v1.4.0
 	github.com/anchore/go-version v1.2.2-0.20200810141238-330bef18dbca
 	github.com/anchore/grype v0.14.1-0.20210702143224-05ade7bbbf70
-	github.com/anchore/syft v0.19.0
+	github.com/anchore/syft v0.22.0
```

Comment:
Where did this `0.22.0` come from, just prepping for the next release?

Reply:
[https://github.com/anchore/syft/releases/tag/v0.22.0](https://github.com/anchore/syft/releases/tag/v0.22.0) Which I see now has not been released. I bumped it to this version specifically to get the addition of the SLES distro type [here](https://github.com/anchore/syft/blob/2f99a35f51dde8c84443ce038b71a748439cf83b/syft/distro/type.go#L20) which, from manual testing, doesn't look like it was in v0.21.0.